temba/flows/models.py

```diff
@@ -5528,6 +5545,9 @@ def execute(self, run, context, actionset_uuid, msg, offline_on=None):
             if self.msg:
                 text = run.flow.get_localized_text(self.msg, run.contact)
 
+            quick_replies = ReplyAction.get_translated_quick_replies(self.quick_replies, run) \
```

Comment:
Don't use ternary statements that span multiple lines - these are easier to read as regular if statements